app/src/main/java/de/westnordost/streetcomplete/quests/opening_hours/AddOpeningHours.kt

```diff
@@ -93,12 +93,25 @@ class AddOpeningHours(
             ),
         ).map { it.key + " ~ " + it.value.joinToString("|") }.joinToString("\n or ") + "\n" + """
             )
-            and !opening_hours
+            and (!opening_hours or opening_hours older today -1 years)
+          )
+          or
+          (
+          opening_hours older today -1 years
+          and
+            (
+            leisure=park
+            or
+            barrier
+            or
+            amenity=toilets
+            or
+            amenity=bicycle_rental
+            )
```

Comment:
```suggestion
          or (
            opening_hours older today -1 years
            and (
              leisure = park
              or barrier
              or amenity ~ toilets|bicycle_rental
            )
```

indentation and formatting